### PR TITLE
Restore `bugs-tab`

### LINKED ISSUE #3360: Bug Tab no longer working
Did not have time to de"bug" it yet

### PR DESCRIPTION
**Pull request by yakov116**
Fixes #3360

### CODE REVIEW

**Comment by fregante** [comment]
I think it used `.js-repo-nav` before too, so we can probably **replace** most `.pagehead` selectors with it.

**Comment by yakov116** [comment]
I will try it now.

**Comment by yakov116** [comment]
We used '[data-hotkey="g i"]' but now there are 2 of them. so we need to specify which one. 

I dont know how the old layout was

**Comment by fregante** [comment]
> We used '[data-hotkey="g i"]' but now there are 2 of them. so we need to specify which one.

Right, but `.js-repo-nav` **is** the right one; the first of those link is in the page header

File: source/features/bugs-tab.tsx [hunk 1]
```diff
@@ -39,7 +39,10 @@ async function init(): Promise<void | false> {
 		return false;
 	}
 
-	const issuesTab = (await elementReady('.pagehead [data-hotkey="g i"]'))?.parentElement;
+	const issuesTab = (await elementReady([
+		'.pagehead [data-hotkey="g i"]',
+		'.js-repo-nav [data-hotkey="g i"]' // "Repository refresh" layout
+	].join()))?.parentElement;
```
Comment (fregante) [inline source/features/bugs-tab.tsx:45]: Even if it doesn't work on GHE, it's fine
```suggestion
	const issuesTab = (await elementReady('.js-repo-nav [data-hotkey="g i"]'))?.parentElement;
```

Comment (fregante) [reply source/features/bugs-tab.tsx:45]: If you feel like it, you can drop `.parentElement` here
```suggestion
	const issuesLink = await elementReady('.js-repo-nav [data-hotkey="g i"]'));
```

And move it to:

```ts
	const bugsTab = issuesTab.parentElement!.cloneNode(true);
```

Comment (busches) [reply source/features/bugs-tab.tsx]: This exists in GHE with the new selector. If there are specific concerns, @ me and I can check.

**Comment by fregante** [comment]
🙌

**Comment by yakov116** [comment]
My 100th commit to refined github!!

**Pull request merged**